inference/nlp/realtime/llm/bloom_176b/djl_deepspeed_deploy.ipynb

```diff
@@ -0,0 +1,613 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.                # "VolumeSizeInGB" : 200,
</pre><p>You don't mention why this is commented out. When should they use it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3642/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Added the comments to the notebook explaining when and how to use it</p>